go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/uor-framework/collection-spec
```

Comment:
Do we care about the org name here?

Reply:
I can update it.

Reply:
Actually, this would require changes to PR https://github.com/emporous/emporous-go/pull/117. Since I have another PR for the rename in that repo, I would prefer to make the change in a second PR.

Reply:
okie dokie. Just checking. lgtm